src/components/katacoda.js

```diff
@@ -57,15 +61,34 @@ const Katacoda = ({account,
             "data-katacoda-externalcss": externalcss, 
             "data-katacoda-hideiframebuttons": hideiframebuttons
         };
+
+        const [shownClass, setShownClass] = useState(clickToShowText ? 'd-none' : '');
+        const scenarioId = account ? [account, scenario].join('/') : scenario;
+
+        // no space reserved when using bottom panel
+        if ( panel ) height = '0';
+
+        const initKata = () => {
+            setShownClass('');
+            if ( panel )
+            {
+                // allow Katacoda to recognize its commands
+                for (var e of document.querySelectorAll(".language-shell")) 
+                    e.dataset.lang = "shell";
+                window.katacoda.init();
+            }
+        };
+
     return <>
         <Helmet>
             <script src="https://katacoda.com/embed.js" />
         </Helmet>
+        <button type="button" onClick={initKata} className={`btn btn-secondary ${!shownClass && 'd-none'}`}>{clickToShowText}</button>
```

Comment:
Unfortunately, this butts into how Katacoda handles "closing" the panel - it destroys it. So while we can re-show it using another call to `window.katacoda.init()`, that also re-creates the environment (e.g., you start over). 

It might be possible to override this behavior & actually just *hide* the panel (similar to how I'm hiding the embedded environment with click-to-show), but it'd require deeper hooks into Katacoda's implementation, and resulting fragility.